src/round-number.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { when, always, compose, lt, applyTo, gt, both } = require('ramda');
+const _curry2 = require('./utils/_curry2');
+const { MAX_MATH_DELTA, MIN_MATH_DELTA, MIN_NEG_MATH_DELTA } = require('./utils/_math-constants');
+
+/**
+ * Rounds a number to the next largest number for positives and the next smallest for negatives
```

Comment:
```suggestion
 * Rounds a given number to the next largest number for positives and the next smallest for negatives.
```

Small wording change.